src/test/java/uk/gov/hmcts/reform/civil/utils/HmcDataUtilsTest.java

```diff
@@ -272,10 +272,10 @@ void getHearingDaysText_shouldReturnExpectedTextForASingle6HourDay() {
     }
 
     @Test
-    void getHearingDaysText_shouldReturnExpectedTextForASingle5HourDay() {
+    void getHearingDaysText_shouldReturnExpectedText_1Day_FullDay() {
```

Comment:
this test is a copy of `getHearingDaysText_shouldReturnExpectedText_1Day_FulDay`

Reply:
on line 259